tests/Oryx.Integration.Tests/DotNetCoreEndToEndTests.cs

```diff
@@ -806,7 +806,7 @@ public async Task CanBuildAndRunApp_WhenOutputIsZipped_AndIntermediateDir_IsUsed
                     "-c",
                     buildImageScript
                 },
-                $"oryxdevms/dotnetcore-{dotnetcoreVersion}",
+                $"oryxdevmcr.azurecr.io/public/oryx/dotnetcore-{dotnetcoreVersion}",
```

Comment:
Using such a long name when working/testing in local environment is really inconvenient. We should have a shorter tag too IMO. For exmample `oryx/build`, `oryx/node-8.1` etc. What do you guys think?

Reply:
I agree. In addition, we should put that name in a variable so that it's easier to change later

Reply:
sure ..  I will do the clean up in a separate work-item I created